rocAL/rocAL/include/meta_data.h

```diff
@@ -28,8 +28,17 @@ THE SOFTWARE.
 #include "commons.h"
 
 
-typedef  struct { float l; float t; float r; float b; } BoundingBoxCord;
+typedef struct BoundingBoxCord_
+{ 
+  float l; float t; float r; float b;
+  BoundingBoxCord_() {}
+  BoundingBoxCord_(float l_, float t_, float r_, float b_): l(l_), t(t_), r(r_), b(b_) {}   // constructor
+  BoundingBoxCord_(const BoundingBoxCord_& cord) : l(cord.l), t(cord.t), r(cord.r), b(cord.b) {}  //copy constructor
+} BoundingBoxCord;
+
+typedef  struct { float xc; float yc; float w; float h; } BoundingBoxCord_xcycwh;
```

Comment:
Instead of creating another Bbox coord structure, please use the BoundingBoxCord structure. You can define the coord as a union to represent either ltrb or xywh format. Also you can add a flag to indicate the mode